src/rules/eol_last.coffee

```diff
@@ -10,4 +10,11 @@ module.exports = class EOLLast
     lintLine: (line, lineApi) ->
         return null unless lineApi.isLastLine()
 
-        return true if line.length
+        isNewline = line.length is 0
+
+        previousIsNewline = if lineApi.lineCount > 1
+            lineApi.lines[lineApi.lineNumber - 1].length is 0
+        else
+            no
```

Comment:
I was going to wait until I decided what to do for a 2.0 version. But I think what will end up happening is 2.0 will enable a lot of rules I wanted on by default but didn't want to break semver.